################################################################################
# PkgEval set-up: 2020-10-30T04:23:11.984
#

Julia Version 1.6.0-DEV.1380
Commit 300e34cf74 (2020-10-30 02:28 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: AMD EPYC 7502 32-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.0 (ORCJIT, znver2)
Environment:
  JULIA_DEPOT_PATH = ::/usr/local/share/julia
  JULIA_MAX_NUM_PRECOMPILE_FILES = 9223372036854775807



################################################################################
# Installation: 2020-10-30T04:23:13.400
#

   Resolving package versions...
   Installed XML2_jll ──────────────────── v2.9.10+3
   Installed Requires ──────────────────── v1.1.0
   Installed Parsers ───────────────────── v1.0.11
   Installed EzXML ─────────────────────── v1.1.0
   Installed JSON ──────────────────────── v0.21.1
   Installed LibPQ ─────────────────────── v1.6.2
   Installed JLLWrappers ───────────────── v1.1.3
   Installed OffsetArrays ──────────────── v1.3.1
   Installed LayerDicts ────────────────── v1.0.0
   Installed Decimals ──────────────────── v0.4.1
   Installed IteratorInterfaceExtensions ─ v1.0.0
   Installed Syslogs ───────────────────── v0.3.0
   Installed IterTools ─────────────────── v1.3.0
   Installed RecipesBase ───────────────── v1.1.0
   Installed ExprTools ─────────────────── v0.1.3
   Installed OpenSSL_jll ───────────────── v1.1.1+6
   Installed LibPQ_jll ─────────────────── v12.3.0+0
   Installed BinaryProvider ────────────── v0.5.10
   Installed Memento ───────────────────── v1.1.1
   Installed DocStringExtensions ───────── v0.8.3
   Installed Zlib_jll ──────────────────── v1.2.11+18
   Installed Infinity ──────────────────── v0.2.3
   Installed DataValueInterfaces ───────── v1.0.0
   Installed Tables ────────────────────── v1.1.0
   Installed DataAPI ───────────────────── v1.3.0
   Installed CEnum ─────────────────────── v0.4.1
   Installed Intervals ─────────────────── v1.5.0
   Installed TableTraits ───────────────── v1.0.0
   Installed Mocking ───────────────────── v0.7.1
   Installed Libiconv_jll ──────────────── v1.16.0+7
   Installed TimeZones ─────────────────── v1.5.0
Updating `~/.julia/environments/v1.6/Project.toml`
  [194296ae] + LibPQ v1.6.2
Updating `~/.julia/environments/v1.6/Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.10
  [fa961155] + CEnum v0.4.1
  [9a962f9c] + DataAPI v1.3.0
  [e2d170a0] + DataValueInterfaces v1.0.0
  [abce61dc] + Decimals v0.4.1
  [ffbed154] + DocStringExtensions v0.8.3
  [e2ba6199] + ExprTools v0.1.3
  [8f5d6c58] + EzXML v1.1.0
  [a303e19e] + Infinity v0.2.3
  [d8418881] + Intervals v1.5.0
  [c8e1da08] + IterTools v1.3.0
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [692b3bcd] + JLLWrappers v1.1.3
  [682c06a0] + JSON v0.21.1
  [6f188dcb] + LayerDicts v1.0.0
  [194296ae] + LibPQ v1.6.2
  [f28f55f0] + Memento v1.1.1
  [78c3b35d] + Mocking v0.7.1
  [6fe1bfb0] + OffsetArrays v1.3.1
  [69de0a69] + Parsers v1.0.11
  [3cdcf5f2] + RecipesBase v1.1.0
  [ae029012] + Requires v1.1.0
  [cea106d9] + Syslogs v0.3.0
  [3783bdb8] + TableTraits v1.0.0
  [bd369af6] + Tables v1.1.0
  [f269a46b] + TimeZones v1.5.0
  [08be9ffa] + LibPQ_jll v12.3.0+0
  [94ce4f54] + Libiconv_jll v1.16.0+7
  [458c3c95] + OpenSSL_jll v1.1.1+6
  [02c8fc9c] + XML2_jll v2.9.10+3
  [83775a58] + Zlib_jll v1.2.11+18
  [0dad84c5] + ArgTools
  [56f22d72] + Artifacts
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [8ba89e20] + Distributed
  [f43a241f] + Downloads
  [7b1f6079] + FileWatching
  [b77e0a4c] + InteractiveUtils
  [b27032c2] + LibCURL
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [a63ad114] + Mmap
  [44cfe95a] + Pkg
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA
  [9e88b42a] + Serialization
  [6462fe0b] + Sockets
  [fa267f1f] + TOML
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode
  [deac9b47] + LibCURL_jll
  [14a3606d] + MozillaCACerts_jll
    Building TimeZones → `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/e863e6524c2a8d703bbd59e7592c97537c94579f/build.log`
    Building LibPQ ────→ `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/12c1a56b0711d60654997c99078a05607e634beb/build.log`


################################################################################
# Testing: 2020-10-30T04:25:29.338
#


     Testing LibPQ
Status `/tmp/jl_YjLmH1/Project.toml`
  [b99e7846] BinaryProvider v0.5.10
  [fa961155] CEnum v0.4.1
  [a93c6f00] DataFrames v0.21.8
  [abce61dc] Decimals v0.4.1
  [ffbed154] DocStringExtensions v0.8.3
  [a303e19e] Infinity v0.2.3
  [d8418881] Intervals v1.5.0
  [c8e1da08] IterTools v1.3.0
  [6f188dcb] LayerDicts v1.0.0
  [194296ae] LibPQ v1.6.2
  [f28f55f0] Memento v1.1.1
  [6fe1bfb0] OffsetArrays v1.3.1
  [bd369af6] Tables v1.1.0
  [f269a46b] TimeZones v1.5.0
  [08be9ffa] LibPQ_jll v12.3.0+0
  [ade2ca70] Dates
  [7b1f6079] FileWatching
  [8f399da3] Libdl
  [8dfed614] Test
Status `/tmp/jl_YjLmH1/Manifest.toml`
  [b99e7846] BinaryProvider v0.5.10
  [fa961155] CEnum v0.4.1
  [324d7699] CategoricalArrays v0.8.3
  [34da2185] Compat v3.23.0
  [9a962f9c] DataAPI v1.3.0
  [a93c6f00] DataFrames v0.21.8
  [864edb3b] DataStructures v0.18.8
  [e2d170a0] DataValueInterfaces v1.0.0
  [abce61dc] Decimals v0.4.1
  [ffbed154] DocStringExtensions v0.8.3
  [e2ba6199] ExprTools v0.1.3
  [8f5d6c58] EzXML v1.1.0
  [a303e19e] Infinity v0.2.3
  [d8418881] Intervals v1.5.0
  [41ab1584] InvertedIndices v1.0.0
  [c8e1da08] IterTools v1.3.0
  [82899510] IteratorInterfaceExtensions v1.0.0
  [692b3bcd] JLLWrappers v1.1.3
  [682c06a0] JSON v0.21.1
  [6f188dcb] LayerDicts v1.0.0
  [194296ae] LibPQ v1.6.2
  [f28f55f0] Memento v1.1.1
  [e1d29d7a] Missings v0.4.4
  [78c3b35d] Mocking v0.7.1
  [6fe1bfb0] OffsetArrays v1.3.1
  [bac558e1] OrderedCollections v1.3.1
  [69de0a69] Parsers v1.0.11
  [2dfb63ee] PooledArrays v0.5.3
  [3cdcf5f2] RecipesBase v1.1.0
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v1.1.0
  [a2af1166] SortingAlgorithms v0.3.1
  [856f2bd8] StructTypes v1.1.0
  [cea106d9] Syslogs v0.3.0
  [3783bdb8] TableTraits v1.0.0
  [bd369af6] Tables v1.1.0
  [f269a46b] TimeZones v1.5.0
  [08be9ffa] LibPQ_jll v12.3.0+0
  [94ce4f54] Libiconv_jll v1.16.0+7
  [458c3c95] OpenSSL_jll v1.1.1+6
  [02c8fc9c] XML2_jll v2.9.10+3
  [83775a58] Zlib_jll v1.2.11+18
  [0dad84c5] ArgTools
  [56f22d72] Artifacts
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8bb1440f] DelimitedFiles
  [8ba89e20] Distributed
  [f43a241f] Downloads
  [7b1f6079] FileWatching
  [9fa8497b] Future
  [b77e0a4c] InteractiveUtils
  [b27032c2] LibCURL
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [44cfe95a] Pkg
  [de0858da] Printf
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics
  [fa267f1f] TOML
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode
  [deac9b47] LibCURL_jll
  [14a3606d] MozillaCACerts_jll
     Testing Running tests...
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:78
  Expression: status(conn) == LibPQ.libpq_c.CONNECTION_OK
   Evaluated: LibPQ.libpq_c.CONNECTION_BAD == LibPQ.libpq_c.CONNECTION_OK
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:78 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:91
  Expression: status(result) == LibPQ.libpq_c.PGRES_TUPLES_OK
   Evaluated: LibPQ.libpq_c.PGRES_FATAL_ERROR == LibPQ.libpq_c.PGRES_TUPLES_OK
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:91 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:92
  Expression: isopen(result)
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:92 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:93
  Expression: LibPQ.num_columns(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:93 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:94
  Expression: LibPQ.num_rows(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:94 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:95
  Test threw exception
  Expression: LibPQ.column_name(result, 1) == "typname"
  BoundsError: attempt to access 0-element Vector{String} at index [1]
  Stacktrace:
   [1] getindex
     @ ./array.jl:787 [inlined]
   [2] column_name(jl_result::LibPQ.Result, column_number::Int64)
     @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/results.jl:455
   [3] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:95 [inlined]
   [4] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [5] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
   [6] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [7] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [8] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [9] top-level scope
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:96
  Expression: LibPQ.column_number(result, "typname") == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:96 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:100
  Test threw exception
  Expression: (data[:typname])[1] == "bool"
  type NamedTuple has no field typname
  Stacktrace:
   [1] getindex(t::NamedTuple{(), Tuple{}}, i::Symbol)
     @ Base ./namedtuple.jl:118
   [2] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:100 [inlined]
   [3] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [4] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
   [5] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [6] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [7] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [8] top-level scope
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:113
  Expression: status(result) == LibPQ.libpq_c.PGRES_TUPLES_OK
   Evaluated: LibPQ.libpq_c.PGRES_FATAL_ERROR == LibPQ.libpq_c.PGRES_TUPLES_OK
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:113 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:114
  Expression: isopen(result)
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:114 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:115
  Expression: LibPQ.num_columns(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:115 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Test Failed at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:116
  Expression: LibPQ.num_rows(result) == 1
   Evaluated: 0 == 1
Stacktrace:
 [1] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:116 [inlined]
 [2] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [3] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
 [4] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [5] macro expansion
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
 [6] macro expansion
   @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
 [7] top-level scope
   @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:117
  Test threw exception
  Expression: LibPQ.column_name(result, 1) == "typname"
  BoundsError: attempt to access 0-element Vector{String} at index [1]
  Stacktrace:
   [1] getindex
     @ ./array.jl:787 [inlined]
   [2] column_name(jl_result::LibPQ.Result, column_number::Int64)
     @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/results.jl:455
   [3] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:117 [inlined]
   [4] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [5] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
   [6] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [7] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [8] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [9] top-level scope
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:121
  Test threw exception
  Expression: (data[:typname])[1] == "bool"
  type NamedTuple has no field typname
  Stacktrace:
   [1] getindex(t::NamedTuple{(), Tuple{}}, i::Symbol)
     @ Base ./namedtuple.jl:118
   [2] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:121 [inlined]
   [3] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [4] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
   [5] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [6] macro expansion
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [7] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [8] top-level scope
     @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
Example SELECT: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:74
  Got exception outside of a @test
  UnknownError: 
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQResultError{LibPQ.Errors.CUN, LibPQ.Errors.EUNOWN})
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_result(jl_result::LibPQ.Result; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/results.jl:228
    [3] prepare(jl_conn::LibPQ.Connection, query::String)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/statements.jl:48
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:129 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:75 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [11] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [12] top-level scope
      @ none:6
   [13] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [14] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [15] _start()
      @ Base ./client.jl:485
Example INSERT and DELETE: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:182
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] LibPQ.Connection(str::String)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:272
    [5] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:183 [inlined]
    [6] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:183 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [12] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [13] top-level scope
      @ none:6
   [14] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [15] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [16] _start()
      @ Base ./client.jl:485
issue #204: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:279
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] LibPQ.Connection(str::String)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:272
    [5] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:280 [inlined]
    [6] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:280 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:279 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [12] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [13] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [14] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [15] top-level scope
      @ none:6
   [16] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [17] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [18] _start()
      @ Base ./client.jl:485
Example COPY FROM: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:303
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] LibPQ.Connection(str::String)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:272
    [5] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:304 [inlined]
    [6] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:304 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:303 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [12] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [13] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [14] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [15] top-level scope
      @ none:6
   [16] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [17] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [18] _start()
      @ Base ./client.jl:485
Wrong column order: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:347
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] LibPQ.Connection(str::String)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:272
    [5] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:348 [inlined]
    [6] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:348 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:303 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [12] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [13] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [14] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [15] top-level scope
      @ none:6
   [16] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [17] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [18] _start()
      @ Base ./client.jl:485
do: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] LibPQ.Connection(f::var"#4#34", args::String; kwargs::Base.Iterators.Pairs{Symbol, Bool, Tuple{Symbol}, NamedTuple{(:throw_error,), Tuple{Bool}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:350
    [5] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:405 [inlined]
    [6] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:403 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [12] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [13] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [14] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [15] top-level scope
      @ none:6
   [16] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [17] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [18] _start()
      @ Base ./client.jl:485
Version Numbers: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:421
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:422 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:422 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Encoding: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:428
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:429 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:429 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Options: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:454
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Symbol, Dict{Symbol, DataType}, Tuple{Symbol}, NamedTuple{(:type_map,), Tuple{Dict{Symbol, DataType}}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:455 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:455 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Time Zone: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:476
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] LibPQ.Connection(f::var"#8#39"{var"#connection_tz#37"}, args::String; kwargs::Base.Iterators.Pairs{Symbol, Bool, Tuple{Symbol}, NamedTuple{(:throw_error,), Tuple{Bool}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:350
    [5] (::var"#7#38"{String, var"#connection_tz#37"})()
      @ Main ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:493
    [6] withenv(f::var"#7#38"{String, var"#connection_tz#37"}, keyvals::Pair{String, Nothing})
      @ Base ./env.jl:161
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:492 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:477 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
   [12] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [13] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [14] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [15] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [16] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [17] top-level scope
      @ none:6
   [18] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [19] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [20] _start()
      @ Base ./client.jl:485
Finalizer: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:577
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] Connection
      @ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:272 [inlined]
    [5] #19
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:579 [inlined]
    [6] iterate
      @ ./generator.jl:47 [inlined]
    [7] _collect(c::UnitRange{Int64}, itr::Base.Generator{UnitRange{Int64}, var"#19#50"{String}}, #unused#::Base.EltypeUnknown, isz::Base.HasShape{1})
      @ Base ./array.jl:677
    [8] collect_similar(cont::UnitRange{Int64}, itr::Base.Generator{UnitRange{Int64}, var"#19#50"{String}})
      @ Base ./array.jl:606
    [9] map(f::Function, A::UnitRange{Int64})
      @ Base ./abstractarray.jl:2268
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:578 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:578 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
   [15] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [16] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [17] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [18] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [19] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [20] top-level scope
      @ none:6
   [21] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [22] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [23] _start()
      @ Base ./client.jl:485
closed socket (EBADF): Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:638
  Got exception outside of a @test
  SystemError: fdopen: Bad file descriptor
  Stacktrace:
    [1] systemerror(p::String, errno::Int32; extrainfo::Nothing)
      @ Base ./error.jl:168
    [2] #systemerror#61
      @ ./error.jl:167 [inlined]
    [3] systemerror
      @ ./error.jl:167 [inlined]
    [4] Base.Libc.FILE(fd::RawFD, mode::String)
      @ Base.Libc ./libc.jl:84
    [5] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:640 [inlined]
    [6] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [7] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:639 [inlined]
    [8] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [9] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:622 [inlined]
   [10] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [11] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:402 [inlined]
   [12] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [13] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [14] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [15] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [16] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [17] top-level scope
      @ none:6
   [18] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [19] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [20] _start()
      @ Base ./client.jl:485
Nulls: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:687 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:687 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Not Nulls: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:770
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:771 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:771 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Tables.jl: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:895
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:896 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:896 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Duplicate names: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:937
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:938 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:938 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Uppercase Columns: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:953
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:954 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:954 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
PQResultError: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:971
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:972 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:972 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Deprecations: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1018
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1019 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1019 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1018 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [15] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [16] top-level scope
      @ none:6
   [17] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [18] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [19] _start()
      @ Base ./client.jl:485
Automatic: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1035
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1036 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1036 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1018 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [15] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [16] top-level scope
      @ none:6
   [17] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [18] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [19] _start()
      @ Base ./client.jl:485
Overrides: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1066
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1067 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1067 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1018 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [15] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [16] top-level scope
      @ none:6
   [17] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [18] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [19] _start()
      @ Base ./client.jl:485
Default Types: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1105
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1106 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1106 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1105 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1018 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [15] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [16] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [17] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [18] top-level scope
      @ none:6
   [19] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [20] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [21] _start()
      @ Base ./client.jl:485
Specified Types: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1245
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1246 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1246 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1105 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1018 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [15] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [16] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [17] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [18] top-level scope
      @ none:6
   [19] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [20] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [21] _start()
      @ Base ./client.jl:485
Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1313
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1314 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1314 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Syntax Errors: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1423
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1424 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1424 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1423 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [15] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [16] top-level scope
      @ none:6
   [17] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [18] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [19] _start()
      @ Base ./client.jl:485
Wrong No. Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1441
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1442 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1442 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1423 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [13] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [14] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [15] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [16] top-level scope
      @ none:6
   [17] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [18] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [19] _start()
      @ Base ./client.jl:485
Interface Errors: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1465
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1466 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1466 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:686 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
No Params, Output: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1482
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1483 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1483 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1482 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Params, Output: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1503
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1504 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1504 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1482 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Basic: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1532
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1533 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1533 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1530 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Parameters: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1554
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1555 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1555 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1530 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Wait in line to complete: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1586
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1587 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1587 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1530 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Cancel: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1615
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1616 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1616 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1530 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Canceled by closing connection: Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1647
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1648 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1648 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1530 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
FDWatcher: bad file descriptor (EBADF): Error During Test at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1679
  Got exception outside of a @test
  could not connect to server: No such file or directory
  	Is the server running locally and accepting
  	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
  Stacktrace:
    [1] error(logger::Logger, exc::LibPQ.Errors.PQConnectionError)
      @ Memento ~/.julia/packages/Memento/9bH5Y/src/loggers.jl:458
    [2] handle_new_connection(jl_conn::LibPQ.Connection; throw_error::Bool)
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:127
    [3] LibPQ.Connection(str::String; throw_error::Bool, connect_timeout::Int64, options::Dict{String, String}, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
      @ LibPQ ~/.julia/packages/LibPQ/LSA5x/src/connections.jl:320
    [4] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1680 [inlined]
    [5] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [6] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1680 [inlined]
    [7] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [8] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:1530 [inlined]
    [9] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [10] macro expansion
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:72 [inlined]
   [11] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [12] top-level scope
      @ ~/.julia/packages/LibPQ/LSA5x/test/runtests.jl:36
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:444
   [14] top-level scope
      @ none:6
   [15] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [17] _start()
      @ Base ./client.jl:485
Test Summary:                                | Pass  Fail  Error  Total
LibPQ                                        |   51    10     39    100
  ConninfoDisplay                            |    4                   4
  Version Numbers                            |   11                  11
  Online                                     |   36    10     39     85
    Example SELECT                           |    9    10      5     24
    Example INSERT and DELETE                |                 1      1
    load!                                    |                 1      1
      issue #204                             |                 1      1
    COPY FROM                                |                 2      2
      Example COPY FROM                      |                 1      1
      Wrong column order                     |                 1      1
    LibPQ.Connection                         |   18            7     25
      do                                     |                 1      1
      Version Numbers                        |                 1      1
      Encoding                               |                 1      1
      Options                                |                 1      1
      Time Zone                              |                 1      1
      Finalizer                              |                 1      1
      Bad Connection                         |   18            1     19
        timeout                              |    2                   2
        closed socket (EBADF)                |                 1      1
        throw_error=false                    |    8                   8
        throw_error=true                     |    8                   8
    Results                                  |    9           15     24
      Nulls                                  |                 1      1
      Not Nulls                              |                 1      1
      Tables.jl                              |                 1      1
      Duplicate names                        |                 1      1
      Uppercase Columns                      |                 1      1
      PQResultError                          |                 1      1
      Type Conversions                       |    9            5     14
        Deprecations                         |                 1      1
        Automatic                            |                 1      1
        Overrides                            |                 1      1
        Parsing                              |    9            2     11
          Default Types                      |                 1      1
          Specified Types                    |                 1      1
          Interval Regex                     |    9                   9
      Parameters                             |                 1      1
      Query Errors                           |                 2      2
        Syntax Errors                        |                 1      1
        Wrong No. Parameters                 |                 1      1
      Interface Errors                       |                 1      1
    Statements                               |                 2      2
      No Params, Output                      |                 1      1
      Params, Output                         |                 1      1
    AsyncResults                             |                 6      6
      Basic                                  |                 1      1
      Parameters                             |                 1      1
      Wait in line to complete               |                 1      1
      Cancel                                 |                 1      1
      Canceled by closing connection         |                 1      1
      FDWatcher: bad file descriptor (EBADF) |                 1      1
ERROR: LoadError: Some tests did not pass: 51 passed, 10 failed, 39 errored, 0 broken.
in expression starting at /home/pkgeval/.julia/packages/LibPQ/LSA5x/test/runtests.jl:34

PkgEval failed: Package LibPQ errored during testing
Stacktrace:
  [1] pkgerror(msg::String)
    @ Pkg.Types /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/Types.jl:52
  [2] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing)
    @ Pkg.Operations /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/Operations.jl:1651
  [3] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:334
  [4] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec})
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:321
  [5] #test#62
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:71 [inlined]
  [6] test
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:70 [inlined]
  [7] #test#61
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:68 [inlined]
  [8] test
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:68 [inlined]
  [9] test(pkg::String; kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:67
 [10] test(pkg::String)
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:67
 [11] top-level scope
    @ none:44
 [12] eval
    @ ./boot.jl:360 [inlined]
 [13] eval(x::Expr)
    @ Base.MainInclude ./client.jl:446
 [14] top-level scope
    @ none:1
 [15] eval(m::Module, e::Any)
    @ Core ./boot.jl:360
 [16] exec_options(opts::Base.JLOptions)
    @ Base ./client.jl:261
 [17] _start()
    @ Base ./client.jl:485


################################################################################
# PkgEval teardown: 2020-10-30T04:26:57.662
CPU usage: 106.43s (99.76s user, 6.93s kernel)

Network usage:
- eth0: 1.687 MiB received, 145.806 KiB sent

Raw statistics: {"blkio_stats":{"io_queue_recursive":[],"io_serviced_recursive":[{"major":259,"value":0,"op":"Read","minor":0},{"major":259,"value":13,"op":"Write","minor":0},{"major":259,"value":0,"op":"Sync","minor":0},{"major":259,"value":13,"op":"Async","minor":0},{"major":259,"value":13,"op":"Total","minor":0},{"major":253,"value":0,"op":"Read","minor":2},{"major":253,"value":13,"op":"Write","minor":2},{"major":253,"value":0,"op":"Sync","minor":2},{"major":253,"value":13,"op":"Async","minor":2},{"major":253,"value":13,"op":"Total","minor":2}],"io_time_recursive":[],"sectors_recursive":[],"io_service_bytes_recursive":[{"major":259,"value":0,"op":"Read","minor":0},{"major":259,"value":73728,"op":"Write","minor":0},{"major":259,"value":0,"op":"Sync","minor":0},{"major":259,"value":73728,"op":"Async","minor":0},{"major":259,"value":73728,"op":"Total","minor":0},{"major":253,"value":0,"op":"Read","minor":2},{"major":253,"value":73728,"op":"Write","minor":2},{"major":253,"value":0,"op":"Sync","minor":2},{"major":253,"value":73728,"op":"Async","minor":2},{"major":253,"value":73728,"op":"Total","minor":2}],"io_wait_time_recursive":[],"io_merged_recursive":[],"io_service_time_recursive":[]},"memory_stats":{"usage":241242112,"max_usage":909193216,"stats":{"hierarchical_memory_limit":9223372036854771712,"pgmajfault":0,"total_rss":210771968,"inactive_file":8192,"inactive_anon":10280960,"total_pgfault":2313668,"total_cache":22679552,"total_rss_huge":0,"total_active_anon":223109120,"total_pgmajfault":0,"pgpgin":1452249,"rss_huge":0,"mapped_file":0,"total_inactive_file":8192,"dirty":0,"total_unevictable":0,"total_dirty":0,"writeback":0,"pgpgout":1395254,"hierarchical_memsw_limit":0,"cache":22679552,"active_file":0,"pgfault":2313668,"total_mapped_file":0,"total_pgpgout":1395254,"total_writeback":0,"rss":210771968,"unevictable":0,"total_inactive_anon":10280960,"total_pgpgin":1452249,"active_anon":223109120,"total_active_file":0},"limit":540972998656},"name":"/LibPQ-9lQIHJh1","networks":{"eth0":{"rx_bytes":1769286,"rx_packets":1274,"tx_packets":1062,"tx_bytes":149305,"tx_dropped":0,"rx_dropped":7,"tx_errors":0,"rx_errors":0}},"id":"7fe30b5db7a3edac5f03950d6427a2928bb3887d1d45296d5b14b565a1824c7c","preread":"0001-01-01T00:00:00Z","storage_stats":{},"precpu_stats":{"throttling_data":{"throttled_periods":0,"throttled_time":0,"periods":0},"cpu_usage":{"usage_in_usermode":0,"usage_in_kernelmode":0,"total_usage":0}},"pids_stats":{"current":38},"num_procs":0,"cpu_stats":{"throttling_data":{"throttled_periods":0,"throttled_time":0,"periods":0},"system_cpu_usage":112846995280000000,"online_cpus":128,"cpu_usage":{"usage_in_usermode":99760000000,"usage_in_kernelmode":6930000000,"total_usage":106425587867,"percpu_usage":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,106425587867,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}},"read":"2020-10-30T04:26:57.664612316Z"}